frontend/app/mocks/db.ts

```diff
@@ -44,6 +44,17 @@ const db = factory({
     referenceId: oneOf('letter'),
     id: primaryKey(faker.string.uuid),
   },
+  country: {
+    code: primaryKey(String),
+    nameEn: String,
+    nameFr: String,
+  },
+  region: {
+    code: primaryKey(String),
+    countryCode: String,
```

Comment:
Consider using a [One-to-One relationship](https://github.com/mswjs/data?tab=readme-ov-file#one-to-one).

Reply:
fixed